src/l500/l500-options.cpp

```diff
@@ -227,22 +361,29 @@ namespace librealsense
                                                   _hw_monitor.get(),
                                                   invalidation_bypass,
                                                   resolution_option.get(),
-                                                  "Enable/disable pixel invalidation" );
+                                                  "Enable/disable pixel invalidation",
+                                                  _fw_version,
+                                                  _digital_gain );
 
-            _digital_gain = register_option<uvc_xu_option<int>, uvc_sensor&, platform::extension_unit, uint8_t, std::string, const std::map<float, std::string>& >
-                (RS2_OPTION_DIGITAL_GAIN, raw_depth_sensor, ivcam2::depth_xu, ivcam2::L500_DIGITAL_GAIN,
-                    "Change the depth digital gain to: 1 for high gain and 2 for low gain",
-                    std::map<float, std::string>{ { RS2_DIGITAL_GAIN_HIGH, "High Gain"},
-                    { RS2_DIGITAL_GAIN_LOW, "Low Gain" }});
+             auto preset = calc_preset_from_controls();
 
+             _preset = std::make_shared< l500_preset_option >(
+                 option_range{ RS2_L500_VISUAL_PRESET_CUSTOM,
+                               RS2_L500_VISUAL_PRESET_SHORT_RANGE,
+                               1,
+                               (float)preset },
```

Comment:
Default should be MAX_RANGE, but the initial value should be `preset`

Reply:
done